gcovr/formats/html/write.py

```diff
@@ -290,7 +290,7 @@ def add_file(
         }
 
         display_filename = force_unix_separator(
-            os.path.relpath(realpath(cdata_fname), self.directory)
+            realpath(cdata_fname)
```

Comment:
Now always absolute paths are displayed instead it relative paths to the root directory.